css/index.css

```diff
@@ -1,3 +1,331 @@
+	margin: 0;
+	padding: 0;
+	border: 0;
+	font-size: 100%;
+	font: inherit;
+	vertical-align: baseline;
+}
+/* HTML5 display-role reset for older browsers */
+article, aside, details, figcaption, figure,
+footer, header, hgroup, menu, nav, section {
+	display: block;
+}
+body {
+	line-height: 1;
+}
+ol, ul {
+	list-style: none;
+}
+blockquote, q {
+	quotes: none;
+}
+blockquote:before, blockquote:after,
+q:before, q:after {
+	content: '';
+	content: none;
+}
+table {
+	border-collapse: collapse;
+	border-spacing: 0;
+}
+
+/* Set every element's box-sizing to border-box */
+* {
+    box-sizing: border-box;
+}
+
+html, body {
+    height: 100%;
+    font-family: 'Titillium Web', sans-serif;
+}
+
+h1, h2, h3, h4, h5 {
+    font-family: 'Bangers', cursive;
+    letter-spacing: 1px;
+    margin-bottom: 15px;
+}
+
+/* Copy and paste your work from yesterday here and start to refactor into flexbox */
+
+
+.container {
+  width: 880px;
+  /* TB LR */
+  margin: 0 auto;
+
+}
+
+
+
+/* header */
+header {
+  display: flex;
+  justify-content: space-between;
+}
+
+header nav {
+	display: flex;
+  flex-flow: row flex-start;
+  justify-content: space-between;
+  align-items: center;
+  margin-top: 20px;
+  width: 70%
+}
+
+header nav a {
+  display: flex;
+  flex-flow: row wrap;
+  font-size: 16px;
+  color: gray;
+  text-decoration: none;
+}
+
+header nav a:hover {
+  text-decoration: underline;
+}
+
+header nav a:first-child {
+  /*margin-left: 0;*/
+}
+
+header nav a:last-child {
+  /*margin-right: 0;*/
+}
+
+header .logo {
+  display: flex;
+  flex-flow: row-reverse;
+
+ }
+
+/* cta */
+.cta {
+  display: flex;
+  margin-top: 50px;
+  flex-flow: row;
+  justify-content: space-around;
+}
+
+.cta .cta-text {
+  display: flex;
+  flex-flow: column;
+  font-size: 77px;
+  text-align: center;
+  align-items: center;
+	align-self: center;
+
+}
+
+.cta .cta-text button {
+  display: flex;
+  justify-content: center;
+  background: white;
+  margin: 10px;
+  font-size: 17px;
+  border: 1px solid black;
+  cursor: pointer;
+  width: 50%;
+}
+
+.cta .cta-text button:hover {
+  background: black;
+  color: white;
+}
+
+.cta img {
+  display: flex;
+  flex-flow: row-reverse;
+
+}
+
+.main-content {
+  border-top: 1px solid black;
+  margin-top: 76px;
+  padding-top: 39px;
+}
+
+.main-content .top-content {
+  display: flex;
+  padding: 0 10px;
+}
+
+}
+
+.main-content .top-content .text-content:last-child {
+  margin-right: 0;
+}
+
+.main-content .middle-img {
+  display: flex;
+  margin-top: 45px;
+}
+
+.main-content .bottom-content {
+  display: flex;
+  margin-top: 37px;
+  padding: 0 10px;
+}
+
+.main-content .bottom-content .text-content {
+
+}
+
+.main-content .bottom-content .text-content:last-child {
+  margin-right: 0;
+}
+
+.contact {
+  margin-top: 57px;
+  border-top: 1px solid black;
+  padding: 39px 10px 0;
+}
+
+.contact p {
+  margin: 20px 0;
+}
+
+footer {
+  margin: 32px 0;
+  text-align: center;
+  font-size: 14px;
+}
+
+/* Tablet view */
```

Comment:
Tablet view looks great! Only a few small comments relating to nit-picky observations between your project and the design file. Mostly what I'm seeing is the spacing between a few elements could take another look.